pennylane/hf/integrals.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the functions needed for computing integrals over basis functions.
+"""
+import numpy as np
+import autograd.numpy as anp
+from scipy.special import factorial2 as fac2
+
+
+def primitive_norm(l, alpha):
+    r"""Compute the normalization constant for a primitive Gaussian function.
+
+    A Gaussian function centred at the position :math:`r = (x, y, z)` is defined as
+
+    .. math::
+
+        G = x^l_x y^l_y z^l_z e^{-\alpha r^2},
```

Comment:
```suggestion
        G = x^{l_x} y^{l_y} z^{l_z} e^{-\alpha r^2},
```
I am not sure it renders correctly without the {}.